examples/simulate_browser.rs

```diff
@@ -137,7 +137,7 @@ fn add_service(client        : std::sync::Arc<std::sync::Mutex<safe_client::clie
     let service_home_dir_name = service_name.clone() + "_home_dir";
 
     let dir_helper = safe_nfs::helper::directory_helper::DirectoryHelper::new(client.clone());
-    let dir_listing = try!(dir_helper.create(service_home_dir_name,
+    let (dir_listing,_) = try!(dir_helper.create(service_home_dir_name,
                                              safe_nfs::UNVERSIONED_DIRECTORY_LISTING_TAG,
```

Comment:
<1> Gap after `,` missing,
<2> You will need to align the next few lines properly.